### ISSUE TITLE
ontology update?

### ISSUE THREAD

**Issue by Antonialock** [pombase/canto#1161]
This term was committed to GO 5 days ago...should be in canto by now?

[Term]
id: GO:1905267
name: endonucleolytic cleavage involved in tRNA processing
namespace: biological_process
def: "Any endonucleolytic RNA phosphodiester bond hydrolysis that is involved in tRNA processing." [GO_REF:0000060, GOC:TermGenie, PMID:25401760]
synonym: "RNA phosphodiester bond hydrolysis, endonucleolytic involved in tRNA maturation" EXACT [GOC:TermGenie]
is_a: GO:0090502 {is_inferred="true"} ! RNA phosphodiester bond hydrolysis, endonucleolytic
intersection_of: GO:0090502 ! RNA phosphodiester bond hydrolysis, endonucleolytic
intersection_of: part_of GO:0008033 ! tRNA processing
relationship: part_of GO:0008033 {is_inferred="true"} ! tRNA processing
created_by: al
creation_date: 2016-06-14T14:54:54Z

**Comment by kimrutherford** [comment]
The ontology load is failing and it's not emailing us about it.  I'll fix it.

**Comment by kimrutherford** [comment]
>  I'll fix it.

I've had a look and I can't work it out what's going wrong.  I have to head to bed now but I'll look again tomorrow.

**Comment by Antonialock** [comment]
is it a problem? Midori sent this to internal:
The term Antonia asked about, GO:1905267, is in the file Canto loads as of
today (looks like it got in on the 18th).

**Comment by ValWood** [comment]
I'm in the GO office. Some terms which got committed got "lost". This is one of them...they are working on it....

**Comment by ValWood** [comment]
OK that isn't the problem, the term is in go.owl and go_plus.owl, so it something else.

**Comment by mah11** [comment]
No, this is not one of the lost terms, at least according to both the email from David OS and the most recent GO files.

GO:1905271-1905273 were listed in the email, and are not in the GO files generated by the release procedure (tho they are in the editors' file).

GO:1905267 is in the released GO files, but hasn't been loaded into Canto.

**Comment by ValWood** [comment]
Yes sorry Tony was checking in QuickGO,rather than GO...sorry.

**Comment by kimrutherford** [comment]
> GO:1905267 is in the released GO files, but hasn't been loaded into Canto.

Yep, there's an error while updating Canto so there isn't a problem with GO.

**Comment by kimrutherford** [comment]
This is fixed now.

The cause was odd.  Canto loads go-svn/ontology/gene_ontology_edit.obo to get GO terms.  I originally used that file because it sounded like the source-of-truth for GO.  But the missing term isn't in gene_ontology_edit.obo - it is in the other OBO files from GO SVN (go.obo, go-basic.obo and go-simple.obo).

I changed the update script to use go-basic.obo and GO:1905267 is now in Canto.

It's not in QuickGO.  Is that unexpected?

For FYPO we use fypo-simple.obo rather than the source file (fypo_edit.obo).  We should probably have switched to using go-basic.obo or go-simple.obo ages ago for Canto but it was working as it was and I didn't want to break things.


> Yep, there's an error while updating Canto so there isn't a problem with GO.

I misread the log files and the error I saw was from ages ago.
